gpt4all-chat/embllm.h

```diff
@@ -1,10 +1,11 @@
 #ifndef EMBLLM_H
 #define EMBLLM_H
 
+#include <QNetworkAccessManager>
```

Comment:
I'm fine with this but the reason I usually put QObject header first is this class inherits it.

Reply:
I always sort blocks of #includes alphabetically. I don't think it's worth thinking about the tree of dependencies - to me that is an implementation detail, no matter the library or framework. We #include QNetworkAccessManager to expose its API, and then QObject to expose the QObject API.